README.md

```diff
@@ -418,6 +418,7 @@ The following settings are optional and allow you to further configure your clus
 * `settings.kubernetes.container-log-max-files`: The maximum number of container log files that can be present for a container.
 * `settings.kubernetes.container-log-max-size`: The maximum size of container log file before it is rotated.
 * `settings.kubernetes.cpu-manager-policy`: Specifies the CPU manager policy. Possible values are `static` and `none`. Defaults to `none`. If you want to allow pods with certain resource characteristics to be granted increased CPU affinity and exclusivity on the node, you can set this setting to `static`. You should reboot if you change this setting after startup - try `apiclient reboot`.
+* `settings.kubernetes.cpu-manager-policy-options`: Policy options to apply when `cpu-manager-policy` is set to `static`. Currently `full-pcpus-only` is the only option.
```

Comment:
Would it make sense to represent this as a list?

```
[settings.kubernetes]
cpu-manager-policy = "static"
cpu-manager-policy-options = [
  "full-pcpus-only",
  "distribute-cpus-across-numa",
  "align-by-socket",
]
```

Versus:
```
[settings.kubernetes]
cpu-manager-policy = "static"

[settings.kubernetes.cpu-manager-policy-options]
full-pcpus-only = true
distribute-cpus-across-numa = true
align-by-socket = true
```

I suppose the drawback is that if we ever needed to add a non-feature-flag option, we'd have to support both with complex serialization logic.

In either case, it'd be good to add an example since this documentation is ambiguous on what the shape should be.

Reply:
Great suggestion - that's actually much better since it allows doing something like:

```
apiclient apply <<EOF
  [settings.kubernetes]
  cpu-manager-policy = "static"
  cpu-manager-policy-options = []
EOF
```

to be able to remove any of these flags, rather than needing to have them included but disabled.

Expanded readme to include this as an example.